### ISSUE TITLE
RxDB dump with seq and last_seq

### ISSUE THREAD

**Issue by mgesmundo** [pubkey/rxdb#1867]
Hello there,
due the latest big big issue with indexeddb on iOS13 (see https://github.com/pouchdb/pouchdb/issues/7928) I switched my app to the `memory` adapter and I backup/restore the whole database on every app bootstrap. Using the RxDB dump and importDump I'm able to do this and it works fine but the dump does not contains the checkpoints for every PouchDB database (the dump does not have any reference to the seq and last_seq). Because of this every sync with the remote endpoint pulls all records again. How I can dump both RxDB data and checkpoints after a whole sync?
Thank you for any suggestion.
All the best

**Comment by pubkey** [comment]
This is not possible with RxDB at the moment.
You could create a PR because I think this is a useful feature.

**Comment by mgesmundo** [comment]
Ok Daniel, I'm working on it.

**Comment by pubkey** [comment]
Any update on this?

**Comment by mgesmundo** [comment]
Hi Daniel,
using the first approach I encountered a problem with the deleted documents: a PouchDB backup can't save the deleted documents. Due this I wrote a PouchDB in-memory adapter (using memdown) that save the in-memory red-black tree to a persistent storage when the red-black tree changes. A generic persistent storage must implement the getItem, setItem, removeItem and an async iterator. The persistent storage for my applications is the Capacitor FileSystem.
I have this solution in test for now but I need to write some documentation. When I put it into the production application I publish it.
